README.md

```diff
@@ -9,19 +9,34 @@ Can be used as a module and from the command line.
 
 See the [CHANGELOG] for details about the latest release.
 
-Features
+# Table of contents
+1. [Features](#features)
+2. [How to install](#howToInstall)
+3. [Usage](#usage)
+    1. [CLI](#cli)
+    2. [Javascript Module](#jsModule)
+4. [json2csv Parser (Synchronous API)](#json2csvSync)
+5. [json2csv Async Parser (Streaming API)](#json2csvAsync)
+5. [json2csv Transform (Streaming API)](#json2csvTransform)
+6. [Upgrading Your Version of JSON2CSV](#upgrading)
+7. [Known Gotchas](#knownGotchas)
+8. [Contributing](#contributing)
 
-- Fast and lightweight
+
+
+## Features <a name="features"></a>
```

Comment:
Why do we need this `<a name="features"></a>`?

Reply:
I added it just for consistency, features made more sense to me as another bullet

Reply:
`## Features` is ok.

What I don't understand is the need for `<a name="features"></a>`